pages/index.tsx

```diff
@@ -2,71 +2,29 @@ import type { NextPage } from "next";
 import Head from "next/head";
 import Image from "next/image";
 import styles from "../styles/Home.module.css";
+import BlogSection from "components/BlogSection/BlogSection";
+import { getPosts } from "@services/contentful";
 
-const Home: NextPage = () => {
+const Home: NextPage = (props: any) => {
```

Comment:
Let's prefer to type anything before merging. 

Don't you think you could use this interface as the collection you expect to receive in this props? https://github.com/MZegarraGlobant/blog-onboarding-exercise-Mike/blob/31a23de9ff03384bf1297457715a63346ea46688/components/Post/Post.tsx#L3

Maybe something like this works:

```ts
interface IHome {
  posts: IPost[]
}
```